x-pack/plugins/security_solution/public/management/cypress/support/e2e.ts

```diff
@@ -50,4 +50,42 @@ Cypress.Commands.addQuery<'findByTestSubj'>(
   }
 );
 
+Cypress.Commands.add(
+  'waitUntil',
+  { prevSubject: 'optional' },
+  (subject, fn, { interval = 500, timeout = 30000 } = {}) => {
+    let attempts = Math.floor(timeout / interval);
```

Comment:
So with default values, the attempts will be `60`.

Reply:
I think so. I copied this from the the other security solution cypress suite

Reply:
Cool. Just wanted to be sure. 😅